aws-logs-resourcepolicy/aws-logs-resourcepolicy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "typeName": "AWS::Logs::ResourcePolicy",
+  "description": "The resource schema for AWSLogs ResourcePolicy",
+  "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs.git",
+  "properties": {
+    "PolicyName": {
+      "description": "A name for resource policy",
+      "type": "string",
+      "pattern": "^([^:*\\/]+\\/?)*[^:*\\/]+$",
+      "minLength": 1,
+      "maxLength": 255
+    },
+    "PolicyDocument": {
+      "description": "The policy document",
+      "type": "string",
+      "pattern": "[\\u0009\\u000A\\u000D\\u0020-\\u00FF]+",
+      "minLength": 1,
+      "maxLength": 131072
+    }
+  },
+  "additionalProperties": false,
+  "primaryIdentifier": [
+    "/properties/PolicyName"
+  ],
+  "createOnlyProperties": [
+    "/properties/PolicyName"
```

Comment:
Don't think we want this here because PolicyName will be used in update operations as well

Reply:
Ok, I've removed all the createOnlyProperties.

Reply:
Now I get a warning:
`Property 'primaryIdentifier' - /properties/PolicyName must be specified as either readOnly or createOnly`.  I'm not sure what to do here.

Reply:
hm good call out. Let me check with CFN team to see what they recommend in this case. Since it's only warning, it might not be breaking behavior, but we should verify this first.

Reply:
Since you can't update a resource policy's PolicyName without creating a new resource policy, we shouldn't expect to need the ability to change PolicyName after creating the resource policy. Having PolicyName be createOnly is the correct move.